### PR TITLE
QA cookie banner rules update

### PR DESCRIPTION
**Pull request by Softvision-PatriciuPop**
Adding 81 new rules for:
swedbank.se, argos.co.uk, gutekueche.at, offnews.bg, coolinarika.com, lidl.cz, ilmessaggero.it, rtp.pt, altex.ro, lidl.sk, filmaffinity.com, tutti.ch, today.ua, telia.fi, vrisko.gr, ripost.hu, ah.nl, dn.pt, dcnews.ro, metoffice.gov.uk, buienradar.be, k-ruoka.fi, cdiscount.com, dikaiologitika.gr, fantacalcio.it, elkjop.no, naszemiasto.pl, nit.pt, halooglasi.com, mall.sk, viaplay.se, ladepeche.fr, virgilio.it, met.ie, instructure.com, amazon.es, antena3.ro, imhd.sk, elcorteingles.es, telegraph.co.uk, vol.at, voetbal24.be, petel.bg, poslovni.hr, lidovky.cz, verkkouutiset.fi, marmiton.org, ethnos.gr, ingatlan.com, dagospia.com, finansavisen.no, gandul.ro, tvnoviny.sk, independent.co.uk, chip.de, hbvl.be, borger.dk, purepeople.com, enikos.gr, uio.no, plotek.pl, fanatik.ro, sydsvenskan.se, zalando.ch, post.at, frognews.bg, lifo.gr, promotions.hu, lastampa.it, rsvplive.ie, capital.ro, 021.rs, telecinco.es, gordonua.com, meteomedia.com, rezultati.com, nbg.gr, tim.it, euro.com.pl, romaniatv.net, mozzartbet.com

### CODE REVIEW

File: cookie-banner-rules-list.json [hunk 1]
```diff
@@ -7476,6 +7476,1182 @@
...
+      "click": {
+        "optIn": "button#onetrust-accept-btn-handler",
+        "presence": "div.banner-actions-container"
+      },
+      "domain": "gutekueche.at",
+      "cookies": {
+        "optIn": [
+          {
+            "name": "_pbjs_userid_consent_data",
+            "value": "618401108686130"
```
Comment (artines1) [inline cookie-banner-rules-list.json:7527]: It looks to me that this value is related to user ID from its name. Maybe we shouldn't include a cookie that could be derived from a user id in our list. @Trikolon wdyt?

File: cookie-banner-rules-list.json [hunk 2]
```diff
@@ -7476,6 +7476,1182 @@
...
+      "cookies": {
+        "optIn": [
+          {
+            "name": "rtp_cookie_privacy",
+            "value": "permit 1,2,3,4"
+          },
+          {
+            "name": "googlepersonalization",
+            "value": "1"
+          }
```
Comment (artines1) [inline cookie-banner-rules-list.json:7598]: Is this cookie really needed to dismiss the banner? Could you verify this? Thanks.

File: cookie-banner-rules-list.json [hunk 3]
```diff
@@ -7476,6 +7476,1182 @@
...
+      "click": {
+        "optIn": "button#didomi-notice-agree-button",
+        "presence": "div#buttons"
+      },
+      "domain": "naszemiasto.pl",
+      "cookies": {
+        "optIn": [
+          {
+            "name": "_pbjs_userid_consent_data",
+            "value": "6798433657233410"
```
Comment (artines1) [inline cookie-banner-rules-list.json:7882]: Same question here. Do we want to include cookies that could be from user id?

**Review by artines1 — COMMENTED** [review]
Nice work to add these rules, really appreciate it.

I have some questions about the list before we can merge this request.

File: cookie-banner-rules-list.json (hunk 1 above)
Comment (Trikolon) [reply cookie-banner-rules-list.json:7527]: Agreed, I'd prefer using a click rule for  this case. It's a sourcepoint frame so this should work:
```
[{
  "domain": "gutekueche.at",
  "click": {
    "optIn": ".sp_choice_type_11",
    "presence": ".message-container >  #notice",
    "skipPresenceVisibilityCheck": true
  },
  "id": "58dd5020-ecc0-4fdc-813b-1009fa83794a"
}]
```

File: cookie-banner-rules-list.json (hunk 3 above)
Comment (Trikolon) [reply cookie-banner-rules-list.json:7882]: imo we shouldn't. This is also prone to break in the future.

File: cookie-banner-rules-list.json (hunk 1 above)
Comment (Softvision-PatriciuPop) [reply cookie-banner-rules-list.json]: Replaced the rule with the one provided by @Trikolon.

File: cookie-banner-rules-list.json (hunk 2 above)
Comment (Softvision-PatriciuPop) [reply cookie-banner-rules-list.json]: Removed the cookie as it was not blocking the banner as intended.

File: cookie-banner-rules-list.json (hunk 3 above)
Comment (Softvision-PatriciuPop) [reply cookie-banner-rules-list.json]: Removed the uuid cookie.

**Review by artines1 — APPROVED** [review]
Thanks for addressing the review comments. It looks good to me now.

**Pull request merged**